smdebug/pytorch/hook.py

```diff
@@ -103,15 +103,18 @@ def _log_params(self, module):
     def _export_model(self):
         pass
 
+    def _get_default_collections(self):
+        return DEFAULT_PYTORCH_COLLECTIONS
+
     def _prepare_collections(self):
-        super()._prepare_collections()
         for coll in self.collection_manager.collections.values():
             for m, (include_inputs, include_outputs) in coll.modules.items():
                 module_name = self.module_maps[m]
                 if include_inputs:
                     coll.include(module_name + "_input_")
                 if include_outputs:
                     coll.include(module_name + "_output_")
+        super()._prepare_collections()
```

Comment:
why this move?

Reply:
`super()._prepare_collections()` checks if include_regex has been set, but this happens in lines 108-116, so we need to move the fn call